ships/0026-image-abstraction-layer.md

```diff
@@ -0,0 +1,232 @@
+---
```

Comment:
The number 26 is already chosen by https://github.com/shipwright-io/community/pull/53. Please use 27, also include a `SHIP-0027: ` prefix in the PR title.

Reply:
Done, thanks for the help.